test/setup/make_source.sh

```diff
@@ -11,3 +13,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+project_id=$(terraform output project_id)
+region=$(terraform output region)
+zone=$(terraform output zone)
+
+{
+  echo "#!/usr/bin/env bash"
+  echo "export TF_VAR_project_id='$project_id'"
+  echo "export TF_VAR_region='$region'"
+  echo "export TF_VAR_zone='$zone'"
+} >> ../source.sh
```

Comment:
It's better to use `>` instead of `>>` for the first entrance, to avoid old vars substitutions

Reply:
@kopachevsky If you use it not once `>>` is bad idea

Reply:
as long it's only place I change file I'll update to >

Reply:
done